exo-setup/src/service-setup.ls

```diff
@@ -9,6 +9,7 @@ require! {
 }
 
 
+# Runs build command in each of the services
 class ServiceSetup extends EventEmitter
```

Comment:
If all testing and running of services is being done inside of docker containers, do we still need this setup process run for each service? 
All it's doing is installing dependencies on the local machine. Those dependencies aren't even copied into the docker image that gets built (due to architecture differences between your local machine and the docker image), the dependencies are installed to the image in the build process itself.

 Speaking of which, currently the Dockerfile runs `npm install --production` whereaas ideally we would be running whatever setup operations are in `service.yml`.

Having this is still helpful if you want to manually run specific tests, but that could be built into `exo-test` in the future.

@kevgo @alexdavid thoughts?